lib/Doctrine/MongoDB/Aggregation/Expr.php

```diff
@@ -41,6 +42,23 @@ class Expr
     private $currentField;
 
     /**
+     * Returns the absolute value of a number.
+     *
+     * The <number> argument can be any valid expression as long as it resolves
+     * to a number.
+     *
+     * @see https://docs.mongodb.org/manual/reference/operator/aggregation/abs/
+     * @param mixed|self $number
```

Comment:
I mainly did that for all other classes as well, we might want to change that. Generally, you can pass:
- A string (denoting a field, like '$foo')
- An expression object (e.g. containing a $sum expression)
- An array (since you could've manually created an expression object)
- An actual number (int or float or MongoInt*) that will be passed to the pipeline.

Back when I wrote the methods I thought this would be the best idea to show what can be passed without specifying each type individually but still showing that you can pass an expression object as well.